packages/block-library/src/video/index.js

```diff
@@ -5,6 +5,7 @@ import { __ } from '@wordpress/i18n';
 import { RichText } from '@wordpress/editor';
 import { createBlock } from '@wordpress/blocks';
 import { createBlobURL } from '@wordpress/blob';
+import { video as videoIcon } from '@wordpress/icons';
```

Comment:
You could probably do this a bit differently:

`import { video as icon } from '@wordpress/icons';`

and then later in the file be able to use `icon,` instead of `icon: videoIcon`.